mmv1/api/type.rb

```diff
@@ -36,13 +36,24 @@ module Fields
       # a different version.
       attr_reader :removed_message
 
-      attr_reader :output # If set value will not be sent to server on sync
-      attr_reader :immutable # If set to true value is used only on creation
+      # If set value will not be sent to server on sync.
+      # For nested fields, this also needs to be set on each descendant (ie. self,
+      # child, etc.).
+      attr_reader :output
+
+      # If set to true value is used only on creation.
```

Comment:
```suggestion
      # If set to true, changes in the field's value require recreating the resource
```

Reply:
The current docs (before your change) describe `ignore_read` more than anything else lol